.travis.yml

```diff
@@ -9,24 +9,78 @@ branches:
 
 matrix:
   include:
-    - python: 3.7
+    - os: linux
+      python: 3.7
       env: NOX_SESSION=check
-    - python: 3.7
+    - os: linux
+      python: 3.7
       env: NOX_SESSION=docs
 
-    - python: 3.6
+    - os: linux
+      python: 3.6
       env: NOX_SESSION=test-3.6
-    - python: 3.7
+    - os: linux
+      python: 3.7
       env: NOX_SESSION=test-3.7
-    - python: 3.8-dev
+
+    - os: linux
```

Comment:
Is specifying `os: linux` actually required? I believe it is the default on Travis CI.